charts/datadog-monitor/values.yaml

```diff
@@ -0,0 +1,66 @@
+global:
+  # Add labels from a parent chart to all manifests.
+  labels: {}
+
+fullnameOverride: ""
```

Comment:
`fullnameOverride` is not needed since this chart produces objects with unique monitor names.
Remember to remove it from the `_helpers.tpl` file as well.